docker/base/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FROM ubuntu:bionic
+
+RUN apt-get update \
+ && apt-get install -y \
+    wget \
+    curl \
+    git \
+    build-essential \
+    ca-certificates \
+    protobuf-compiler \
+    python-protobuf \
+    libprotobuf-dev \
+    libssl-dev \
+    libcurl4-openssl-dev \
+    libtool \
+    cmake \
+    vim \
+ && apt-get -y -q upgrade \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+ARG GO_VERSION=go1.13
```

Comment:
For clarity, it would be nicer next to the GO_TAR definition.  Also, i guess right now we need go really only for dev? So why not just put it there?

On the other hand, right now we have sgxssl only in dev, once we would add deterministic build we would need it also in ``ccenv-build'' and so might need it in two different docker images?

Reply:
ok, let's move go to dev image for the moment.